operations/helm/charts/mimir-distributed/values.yaml

```
@@ -863,6 +874,8 @@ ingester:
       nodeSelector: null
       # -- extraAffinity adds user defined custom affinity rules (merged with generated rules)
       extraAffinity: {}
+      # -- Ingester data Persistent Volume Storage Class
```

```suggestion
      # -- Ingester data Persistent Volume Storage Class
      # If defined, storageClassName: <storageClass>
      # If set to "-", then use `storageClassName: ""`, which disables dynamic provisioning
      # If undefined or set to null (the default), then fall back to the value of `ingester.persistentVolume.storageClass`.
```

just to be very explicit about what things do. Can you also make the change to the other 2 zones of the ingester? And then for the 3 zones of the store-gateway and alertmanager (modifying the message slightly of course :) )

---

done, thanks.